packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md

```diff
@@ -1,16 +1,9 @@
 The `OktaAuthGuard` can protect a route to a single component as well with `canActivate` property.
 
-
-1. Import the `ProfileComponent` component into `app-routing.module.ts`:
-
-   ```ts
-   import { ProfileComponent } from './profile/profile.component';
-   ```
-
-2. Add the following to the `routes` array:
+1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:
 
    ```ts
    { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
    ```
 
-This single `/profile` route is now protected.
+The single `/profile` route is protected directly, and on launch goes directly to the sign-in page (no need to click the sign-in button).
```

Comment:
I'm not sure I understand this. Would you mind clarifying for me?

Reply:
Oh gotcha. I see what you mean. Would something like this work?

> The single `/profile` route is protected directly, and on navigation will redirect you to the sign-in page if you aren't already authenticated.

Reply:
Perfect. Thanks!